editor-components/feature/feature12/feature12.tsx

```diff
@@ -88,22 +173,77 @@ class Feature12 extends BaseFeature {
   }
 
   render() {
+    const upperTitle = this.getPropValue("upperTitle");
+    const title = this.getPropValue("title");
+    const titleBackingText = this.getPropValue("titleBackingText");
+    const itemCount = this.getPropValue("itemCount");
+    const messageCards = this.castToObject<IMessages[]>("messageCard");
+    const firstItem = this.getPropValue("message");
+    const firstCardTitle = firstItem[0].value;
+    const buttonText = firstItem[1].value;
+    const backgroundImage = firstItem[2].value;
+   
     return (
-      <div className={this.decorateCSS("container")}>
-        <div className={this.decorateCSS("max-content")}>
+      <div className={this.decorateCSS("container")} style={{ paddingBottom: "0px" }}>
+        <div className={this.decorateCSS("upper-title")} style={{ fontSize: "1.2em", textAlign: "center" }}>
+          {upperTitle}
+        </div>
+        <div className={this.decorateCSS("shadow-header-title")} style={{ fontSize: "12.5em", textAlign: "center", marginBottom: "20px" }}>
+          {titleBackingText}
+        </div>
```

Comment:
don't use inline css. move them into the scss file.
don't render relevant divs if the prop is empty.
[see here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)